src/assets/i18n/en.json

```diff
@@ -0,0 +1,472 @@
+    "my": "My groups",
+    "empty": "You currently don't have access to any groups.",
+    "org_item": {
+      "project_count": "| {{ count }} | Projects"
+    },
+    "invites": {
+      "title": "Group invites",
+      "join": "Join group",
+      "empty": "Not invited to any groups",
+      "accepted": "Accepted invitation to organization."
+    },
+    "new": {
+      "title": "Create a new investment group",
+      "title.back": "New group",
+      "logo_input": {
+        "title": "Add logo"
+      },
+      "group_name_input": {
+        "title": "Group name",
+        "placeholder": "Enter group name"
+      },
+      "description_input": {
+        "title": "Description",
+        "placeholder": "Enter description"
+      },
+      "submit_button": "Create new group",
+      "submit_button.loading": "Creating group...",
+      "created": "Investment group created!"
+    },
+    "details": {
+      "title.back": "Investment group details",
+      "create_project": "Create project",
+      "project_count": "| {{ count }} | Projects",
+      "wallet_not_verified": "This group is pending approval from admin. During this time, you can't create new projects.",
+      "wallet_missing": "The organization wallet needs to be created. You will be prompted now.",
+      "projects": {
+        "title": "Projects",
+        "empty": "There are no projects associated with this organization.",
+        "item": {
+          "period": "Funding | {{ from }} - {{ to }}",
+          "raised": "{{ raised }} | raised of {{ goal }} goal",
+          "publish": "Publish",
+          "unpublish": "Unpublish"
+        }
+      },
+      "members": {
+        "title": "Members",
+        "add_input.placeholder": "joe@mail.com, jane@mail.com",
+        "add_button": "Add members",
+        "add_button.loading": "Sending invites...",
+        "join_date": "Joined {{ date }}",
+        "empty": "You are the only member of {{ org }}",
+        "invited": "Successfully invited user to organization.",
+        "deleted": "Successfully deleted user from the organization."
+      }
+    }
+  },
+  "projects": {
+    "new": {
+      "title.back": "Back",
+      "title": "Create a new project",
+      "name_input": {
+        "title": "Project name",
+        "placeholder": "Enter project name"
+      },
+      "period_input": {
+        "title": "Date period (Start - End)",
+        "placeholder.start": "Enter start date",
+        "placeholder.end": "Enter end date"
+      },
+      "investment_range_input": {
+        "title": "Investment per member (Min - Max)",
+        "placeholder.start": "Enter minimal amount",
+        "placeholder.end": "Enter maximal amount"
+      },
+      "expected_funding_input": {
+        "title": "Expected funding",
+        "placeholder": "Enter expected funding amount"
+      },
+      "lock_notice": "Date, Investment per member and Expected funding cannot be changed once project is created.",
```

Comment:
"Date, Investment per member, and Expected funding cannot be changed once **the** project is created."